test/fixtures/desks.yml

```diff
@@ -5,6 +5,7 @@
 # below each fixture, per the syntax in the comments below
 #
 desk1:
+  id: 1
```

Comment:
Shouldn't rails auto-assign this?

Reply:
You're right! I'm not sure why I thought it wouldn't 😅